### ISSUE TITLE
Please fix .gitattributes file

### ISSUE THREAD

**Issue by kloczek** [michaelrsweet/pappl#181]
Please remove line `configure.ac export-ignore` because in autogenerated tar ball there is no `configure.ac`.
After correcting that line it wil be necessary to move git tag.
Also files like 
- Makefile
- config.guess
- config.h.in
- config.sub
- configure
- install-sh

shoud be removed from git repo because they are generated by `autoreconf -fiv` command or `configure` script.

**Comment by michaelrsweet** [comment]
@kloczek i won’t be re-releasing the current beta tarball.

I’m happy to keep configure.ac in the tarball. The other files must remain because I don’t use autoreconf or the other autotools files because they end up changing a lot more files (and in some case break them) than they should. “autoconf -f” is the only thing I do to regenerate the configure script, and I’d even like to eliminate that…

(Keep in mind that a *lot* of developers don’t have the complete GNU tools installed/available so I try to make the gig sources work out of the box…)

**Comment by kloczek** [comment]
I see that you've added this to v1.1 milestone.
Looks like you are using as well a bit messy tagging. Issue is that `v1.1` is lower than latest `v1.1b1` so it will beproblem with upgrade from `1.1b1` to `1.1` using most of the packaging software.

**Comment by michaelrsweet** [comment]
@kloczek As 1.1b1 is a BETA and thus NON STABLE release, it shouldn’t be released as a package. Anyone that does can munge the version number however they need to ensure that 1.1b1 < 1.1.0 (the first stable 1.1 release). Semantic versioning says that I should use a dash (1.1-b1) but RPM wants a tilde (1.1~b1) and other packaging systems use their own syntax as well.

TL;DR: Since pre-release versioning is a mess and I’m just providing a tarball for people to “kick the tires”, I’m sticking with a format that has worked for me for the last 30 years.

**Comment by michaelrsweet** [comment]
@kloczek I'm not going to get into a debate over it. And 30 years ago I wrote EPM to automate packaging across a dozen OS's and packaging systems, complete with a flexible versioning syntax that still works today (and is the reason I use the format I do). FYI, RPM and Debian (as two examples) do NOT support Semantic Versioning, so "1.1.0-beta1" would be treated as NEWER than "1.1.0". Snapcraft doesn't give a shit (everything is based on build number).

**Comment by michaelrsweet** [comment]
[master 4548492] Don't hide configure.ac in tarballs (Issue #181)
